ludwig/features/sequence_feature.py

```diff
@@ -70,40 +69,55 @@ class SequenceFeatureMixin(object):
 
     @staticmethod
     def get_feature_meta(column, preprocessing_parameters):
-        idx2str, str2idx, str2freq, max_length = create_vocabulary(
-            column, preprocessing_parameters['tokenizer'],
+        tokenizer = get_tokenizer(
```

Comment:
I'd like to work on this PR.
What should this functions look like ?

Reply:
Thank you for being willing to help, much appreciated!
Another contributor submitted a PR here https://github.com/uber/ludwig/pull/831#pullrequestreview-468793135, as soon as I merge it (ideally later today), it will be reflected here.
There will still be some work to do after i merge that PR in the hf_transformers branch, specifically testing. Would you be able to help on that? Specifically, add an integration test module like `test_huggingface.py` that is similar to `text_experiment.py` and tests all new encoders on classification, tagging and nlu? I can give more precise instruction if this is too vague.